microcosm_flask/conventions/swagger.py

```diff
@@ -56,12 +56,15 @@ def discover():
     name="swagger",
```

Comment:
We need these changes so that the new operations are mapped into the swagger doc.